### DISCUSSION TITLE
How to programmatically change the thumbnail color?

### DISCUSSION THREAD

**Discussion by Turtleted21** [mdbassit/Coloris#115] [Q&A] [answered]
Hi
How to programmatically change the thumbnail color (without a click/pick event)?

I try this without success :

```
      Coloris({
        el: ".coloris",
        input: "#ff0000",
        wrap: true,
        // swatchesOnly: true,
        alpha: false,
        swatches: ["#264653", "#2a9d8f", "#e9c46a", "#f4a261", "#e76f51", "#48cae4"],
      });
```

default with a empty value :
`<input id="color-changer" type="text" class="coloris" value="" style="width: 25px; height: 25px; border-radius: 5px; box-sizing: border-box" />`

try to change to pink :
`document.getElementById("color-changer").dispatchEvent(new Event("input", { value: "#ff00ff" }));`

or 

`document.getElementById("color-changer").value = "#ff00ff";`

Thanks

**Comment by mdbassit** [comment, marked as answer]
Hello,

First you need to update the color value in the input field:
```js
document.getElementById("color-changer").value = "#ff00ff";
```
Then trigger an input event on the field:
```js
document.getElementById("color-changer").dispatchEvent(new Event("input", { bubbles: true }));
```
